cpp/client/ssl_client.cc

```diff
@@ -201,10 +209,8 @@ int SSLClient::VerifyCallback(X509_STORE_CTX* ctx, void* arg) {
     }
     // Else look for the proof in a superfluous cert.
     // Let's assume the superfluous cert is always last in the chain.
```

Comment:
Done. Split it into two parts, one where we get the value and one where the decision occurs.